NewForce-Cohort-8/tabloidfullstack-the-body-slam-batallion
# Ja post details 5

I completed the View post details feature 

As a reader, I would like to see the content of a Post so I can read it.

Given a user is viewing a list of Posts
When they select a post to read
Then they should be directed to a Post Detail page that shows the Post Details.

Post Details include:

Title
Header image (if exists)
Content
Publication date (MM/DD/YYYY)
Author's Display Nameshould be in order of creation date with the most recent on top

Fixes # (issue)
Ticket #5 https://trello.com/c/zzFmFgwP/5-5-post-details

Type of change
New feature (non-breaking change which adds functionality)

Testing Instructions

1 .After making sure you have our repo cloned down into a local repository
and have the database added, git fetch --all in the master branch to retrieve this branch named JA-postDetails-5 that I have pushed up.

2.git checkout JA-postDetails-5 to switch to my branch and open up Visual Studio.

3.create a new database and run the following Tabloid sql script using SQL Server Object Explorer inside Visual Studio :
-Open a new SQL query
-Copy the following code into the new SQL query and execute it to create database :
Follow this link .https://github.com/NewForce-Cohort-8/tabloidfullstack-the-body-slam-batallion/blob/main/SQL/01_Tabloid_Create_DB.sql

4.Open a new SQL query for your database,
Copy the following code into the new SQL query and execute it to seed your database with the data posted in this SQL Query: Follow this link .https://github.com/NewForce-Cohort-8/tabloidfullstack-the-body-slam-batallion/blob/main/SQL/02_Tabloid_Seed_Data.sql

5.Then, run the application in a browser window. You should see the 'Log in' page with a navigation bar at the top of the page. If not already logged out, log out (with the link in the navbar), and 'Log in' with the following random user credentials.
email: gteanby6@craigslist.orgx

6.Once you're logged in, click on "MyPosts" in the nav bar.

7. You will be directed to a "my post List" page . 
 where you can view the existing user posts and available info and blue details hyperlink .
8. click on details link in post list , you should be directed to a new page with spefic post details for th post you chose.


Checklist
I have opened JA-postDetails-5 in my local repository and made sure I have the SQL database.
 I have ran my application in a browser window to see a 'Log in' page
 After logging in the appropriate credentials, selecting myposts in the navbar, directs me to a "my posts" page.
clicking on deatils link redirects me to details post page
